.github/workflows/ci-owasp-dependency-check-2.7.yaml

```diff
@@ -0,0 +1,62 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: CI - OWASP Dependency Check - Branch 2.7
+on:
+  schedule:
```

Comment:
Why not trigger this only when there has a PR to the branch or push code to the branch?

Reply:
@zymap  The CI capacity isn't infinite. That's one reason to run this only as a scheduled job. Security issues will be discovered independently from the commits. That's another reason to run scheduled jobs at least once per day.